tests/components/switcher_kis/test_services.py

```diff
@@ -105,24 +106,21 @@ async def test_set_auto_off_service_fail(
     entity_id = f"{SWITCH_DOMAIN}.{slugify(device.name)}"
 
     with patch(
-        "homeassistant.components.switcher_kis.switch.SwitcherApi.set_auto_shutdown",
+        "homeassistant.components.switcher_kis.entity.SwitcherApi.set_auto_shutdown",
         return_value=None,
     ) as mock_set_auto_shutdown:
-        await hass.services.async_call(
-            DOMAIN,
-            SERVICE_SET_AUTO_OFF_NAME,
-            {ATTR_ENTITY_ID: entity_id, CONF_AUTO_OFF: DUMMY_AUTO_OFF_SET},
-            blocking=True,
-        )
+        with pytest.raises(HomeAssistantError):
+            await hass.services.async_call(
+                DOMAIN,
+                SERVICE_SET_AUTO_OFF_NAME,
+                {ATTR_ENTITY_ID: entity_id, CONF_AUTO_OFF: DUMMY_AUTO_OFF_SET},
+                blocking=True,
+            )
 
         assert mock_api.call_count == 2
         mock_set_auto_shutdown.assert_called_once_with(
             time_period_str(DUMMY_AUTO_OFF_SET)
         )
-        assert (
-            f"Call api for {device.name} failed, api: 'set_auto_shutdown'"
-            in caplog.text
-        )
```

Comment:
Please remove the unused `caplog`

Reply:
It is done.

Reply:
Remove the `caplog` fixture as it is not used https://github.com/home-assistant/core/blob/1aad3f8a0f9e070e129f9cdc2a4582c07a5cb00b/tests/components/switcher_kis/test_services.py#L99

Reply:
It is done.